### DISCUSSION TITLE
Android App

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#800] [Feature Requests]
I have been using the Chrome browser and the "Add To Home Screen" option on Android devices as a substitute for the old Rune app. It now looks as if all mobile browser apps are losing the option to add a shortcut which will open rAudio in full-screen mode.

Is there any chance of a new app, or is there another way?

**Comment by ceejayemm** [comment]
@R101G 

The 'old' Android app works just fine with rAudio-1, I use it constantly, the only thing is you need to manually add your individual systems rather than allow the app to discover them.

Chris

**Comment by R101G** [comment]
@ceejayemm 

Yes, I am using it, but it's a bit untidy. Wrong logo, and it sometimes scrolls the metadata when first opened. Also, as you say, you have to enter an IP rather than have it search for the host name. 

It would be nice to have an rAudio version.

**Reply by elliotn** [reply]
Not sure what rern has planned, but I tried to explain how to make it work here: https://github.com/rern/rAudio-1/discussions/746

**Comment by hjfisch** [comment]
Also works with native Android-Emu in Windows 11

**Comment by R101G** [comment]
I have no problem with using the IP, but the Android Chrome "add to home screen" showed the right logo, was full-screen, and had no display bugs. Now that this feature is disappearing, it would be good to have a dedicated app which rern is in control of.

**Reply by rern** [reply]
Please try: [rAudio app - Android](https://github.com/rern/rern.github.io/releases/download/20210307/rAudio-20220214.apk)
(Allow installation of apps from unknown sources - enable > install > disable)

**Reply by klassikmann** [reply]
@rern 
My subnet is 192.168.**2** not 192.168.**1**. It is not  easy to change because "." can't be entered.

**Comment by hjfisch** [comment]
App is working , but it would be nice if I could change the ip inside , because I've two different RPI running with rAudio . So I need to close the app in the Taskmanager and then to restart to change the IP .

**Reply by rern** [reply]
Next update will do.

**Comment by rern** [comment]
Now [rAudio app](https://play.google.com/store/apps/details?id=com.raudio) is up at Google Play Store.

**Comment by Raitsa** [comment]
Great work again 👍
Installed on 2 android phones and one android tablet, all systems go 🕺

**Comment by R101G** [comment]
@rern 

Thanks rern, that's brilliant.

The only bug I have found is that whenever the app is cleared and re-opened, I have to re-enter the IP address.

**Reply by rern** [reply]
rAudio app saves the latest IP address to `SharedPreferences` on the device and retrieved on each start. Clear app data also clears the saved IP.

**Reply by R101G** [reply]
@rern 
I am using an Android 11 tablet, and this does not seem to be working. If I use another app and go back to rAudio, I have to enter the IP again. I don't know where to look for SharedPreferences.

**Comment by kosto-stavr** [comment]
Every time you start the app, you have only to press OK to the popup message.
The IP (in the popup message ) remains the same with the last used (because  it kept in SharedPreferences...)

**Comment by R101G** [comment]
I find that just rotating the tablet  90 degrees  will bring up an IP entry screen showing the correct IP, so it is being saved somewhere but still has to be re-entered constantly. I suspect this has something to do with the complicated permissions which Android 11 has.

**Comment by Raitsa** [comment]
rAudio app updated, no more hassles with the ip.

Improve startup:
- Verify IP address - valid and reachable
- Auto connect - with last connected IP address

**Comment by hjfisch** [comment]
I need a direct link to the app , without going to the google appstore.

**Comment by Raitsa** [comment]
https://apkpure.com/raudio/com.raudio

Not the latest one but I suspect updates will show up there too.

**Comment by R101G** [comment]
App updated automatically, and now retains the IP setting. Thanks.

**Comment by Sigmun** [comment]
How to access to settings within the app? I wish I could activate snapcast client from my phone.
